[2022-08-19 23:12:01,797] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T07:59:00+00:00 [queued]>
[2022-08-19 23:12:01,813] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T07:59:00+00:00 [queued]>
[2022-08-19 23:12:01,815] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:12:01,816] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:12:01,820] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:12:01,833] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 07:59:00+00:00
[2022-08-19 23:12:01,838] {standard_task_runner.py:52} INFO - Started process 4980 to run task
[2022-08-19 23:12:01,841] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T07:59:00+00:00', '--job-id', '978', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpp6ugo0rt', '--error-file', '/tmp/tmpg283_o2e']
[2022-08-19 23:12:01,842] {standard_task_runner.py:80} INFO - Job 978: Subtask get_telegram_version
[2022-08-19 23:12:01,929] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T07:59:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:12:01,988] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T07:59:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T07:59:00+00:00
[2022-08-19 23:12:01,990] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:12:02,009] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T075900, start_date=20220819T231201, end_date=20220819T231202
[2022-08-19 23:12:02,055] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:12:02,076] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
